### ISSUE TITLE
Improvement: Support binary file generation in lein-new

### ISSUE THREAD

**Issue by sbondaryev** [technomancy/leiningen#2021]
Currently we can't generate binary files (images etc.) with 'renderer' function.
So lots of clojure web-projects use their own implementation like:
```clj
(defn resource [r]
(->> r (str "leiningen/new/cryogen/") (io/resource) (io/input-stream)))
```
Could it be useful to add this function in templates.clj?

**Comment by hyPiRion** [comment]
Yeah, that wouldn't be a bad addition. Happy to take a PR for that.

**Comment by sbondaryev** [comment]
I replaced default render io/reader by io/input-stream
is it a right solution?

**Comment by hyPiRion** [comment]
Fixed by #2022.
